android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -269,6 +272,7 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
         view.addOnAttachStateChangeListener(object : android.view.View.OnAttachStateChangeListener {
             override fun onViewAttachedToWindow(v: android.view.View?) {}
             override fun onViewDetachedFromWindow(v: android.view.View?) {
+                fetchResJob?.cancel()
```

Comment:
destroyModel() is called below which will call cancel on the job, is it necessary to cancel here as well?